source/backend/api/Areas/Reports/Mapping/Lease/LeaseMap.cs

```diff
@@ -25,7 +25,7 @@ private static void MapLease((Entity.PimsLeasePeriod period, Entity.PimsLease le
             var historicalString = src.property?.Property?.GetHistoricalNumbersAsString();
             var currentRenewal = src.lease.PimsLeaseRenewals.FirstOrDefault(renewal => renewal != null && renewal.IsExercised == true && DateTime.Now > renewal.CommencementDt && DateTime.Now <= renewal.ExpiryDt);
 
-            var additionalRenewalsCount = src.lease.PimsLeaseRenewals.FirstOrDefault(r => r.LeaseRenewalId == currentRenewal.LeaseRenewalId) == null ? src.lease.PimsLeaseRenewals.Count : src.lease.PimsLeaseRenewals.Count - 1;
+            var additionalRenewalsCount = src.lease.PimsLeaseRenewals.FirstOrDefault(r => r.LeaseRenewalId == currentRenewal?.LeaseRenewalId) == null ? src.lease.PimsLeaseRenewals.Count : src.lease.PimsLeaseRenewals.Count - 1;
```

Comment:
currentRenewal is populated by FirstOrDefault on line 26, meaning it can be null, so dereferencing without a null check will cause this to fail.